src/main/java/com/greenfoxacademy/domain/GenericToken.java

```diff
@@ -23,52 +23,42 @@
     @Temporal(value = TemporalType.TIMESTAMP)
     Date valid;
 
+
     @Transient
-    final int DEFAULT_TIMEOUT = 30;
+    final int DEFAULT_TIMEOUT = 30*60*1000;
```

Comment:
@viktornagy there should be a constant for minute in millisec, what you should use later...